Sources/App/Views/Keyword/KeywordShow+View.swift

```diff
@@ -66,8 +79,16 @@ enum KeywordShow {
                 )
             )
         }
-    }
 
+        private func showPackageLimitDisclaimer() -> Node<HTML.BodyContext> {
+            guard model.totalPackageCount >= Constants.maxKeywordPackageCollectionCount else { return .empty }
```

Comment:
I think this should be a simple greater than rather than greater than or equal to. We say it will be limited to 300 in the note.